CODEOWNERS

```
@@ -1,9 +1,8 @@
 # Global owners
 
-- @cloudflare/wrangler
-  /packages/wrangler/CHANGELOG.md @cloudflare/wrangler-admins
-  /packages/pages-shared/ @cloudflare/pages
-  /packages/wrangler/pages/ @cloudflare/pages
-  /packages/wrangler/src/pages/ @cloudflare/pages
-  /packages/wrangler/src/pages.\* @cloudflare/pages
-  /packages/create-cloudflare/ @jculvey @RamIdeas @petebacondarwin
+* @cloudflare/wrangler
+/packages/wrangler/CHANGELOG.md @cloudflare/wrangler-admins
+/packages/pages-shared/ @cloudflare/pages
+/packages/wrangler/pages/ @cloudflare/pages
+/packages/wrangler/src/pages/ @cloudflare/pages
```

```suggestion
/packages/wrangler/src/pages/ @cloudflare/pages
/packages/wrangler/src/d1/ @cloudflare/d1
```
Can we get code ownership over the D1 folder too?
CC @elithrar

---

Sure, sounds good